content/applications/hr/payroll/contracts.rst

```diff
@@ -89,6 +89,17 @@ Contract details
 The contract details section allows for the addition and editing of a contract, and the ability to
 send the contract to the employee for approval and signatures.
 
+.. important::
+   To access the various contract template fields in the :guilabel:`Contract Details` section, the
+   *Salary Configurator* module **must** be installed.
+
+   To install this module, navigate to the :menuselection:`Apps` application. In the search bar,
```

Comment:
```suggestion
   To install this module, navigate to the :menuselection:`Apps` application. In the :guilabel:`Search...` bar,
```

Reply:
I edited this sectoin a bit to match another edit Sam made in a different doc that addresses the same issue.